examples/CSVImporter/CSVImporter.go

```diff
@@ -199,11 +215,10 @@ func ProcessCSVRecord(jc jcapi.JCAPI, userList []jcapi.JCUser, csvRecord []strin
 
 			// Create or modify the tag in JumpCloud
 			tempTag.Id, err = jc.AddUpdateTag(opCode, tempTag)
-
 			if err != nil {
-				fmt.Printf("Could not process tag '%s', err='%s'", tempTag.ToString(), err)
+				err = fmt.Errorf("Could not POST tag '%s', err='%s'", tempTag.ToString(), err)
 			} else {
-				fmt.Printf("Processed tag ID '%s'\n", tempTag.Id)
+				fmt.Printf("\tCreated tag '%s' (ID '%s')\n", tempTag.Name, tempTag.Id)
```

Comment:
We have this in a few places, but why `fmt.Printf("\t.....")`? Wouldn't we want logs on a new line?